src/orders/views/OrderList/filters.ts

```diff
@@ -16,49 +17,52 @@ import {
   getGteLteVariables,
   getMinMaxQueryParam,
   getMultipleEnumValueQueryParam,
+  getMultipleValueQueryParam,
   getSingleValueQueryParam
 } from "../../../utils/filters";
 import {
   OrderListUrlFilters,
   OrderListUrlFiltersEnum,
+  OrderListUrlFiltersWithMultipleValues,
   OrderListUrlFiltersWithMultipleValuesEnum,
   OrderListUrlQueryParams
 } from "../../urls";
 
 export const ORDER_FILTERS_KEY = "orderFilters";
 
 export function getFilterOpts(
-  params: OrderListUrlFilters
+  params: OrderListUrlFilters,
+  channels: MultiAutocompleteChoiceType[]
 ): OrderListFilterOpts {
   return {
+    channel: channels
+      ? {
+          active: params?.channel !== undefined || false,
+          value: channels
+        }
+      : null,
     created: {
-      active: maybe(
-        () =>
-          [params.createdFrom, params.createdTo].some(
-            field => field !== undefined
-          ),
-        false
-      ),
+      active:
+        [params?.createdFrom, params?.createdTo].some(
+          field => field !== undefined
+        ) || false,
       value: {
-        max: maybe(() => params.createdTo, ""),
-        min: maybe(() => params.createdFrom, "")
+        max: params?.createdTo || "",
+        min: params?.createdFrom || ""
       }
     },
     customer: {
-      active: !!maybe(() => params.customer),
-      value: params.customer
+      active: !!params?.customer,
+      value: params?.customer
     },
     status: {
-      active: maybe(() => params.status !== undefined, false),
-      value: maybe(
-        () =>
-          dedupeFilter(
-            params.status.map(status =>
-              findValueInEnum(status, OrderStatusFilter)
-            )
-          ),
-        []
-      )
+      active: params?.status !== undefined || false,
+      value:
+        dedupeFilter(
+          params?.status?.map(status =>
+            findValueInEnum(status, OrderStatusFilter)
+          )
+        ) || []
```

Comment:
Dedupe flter always returns array, so `|| []` not needed, but seems like it will return `[undefined]` if params?.status? is undefined. Maybe worth checking what happens then

Reply:
It looks like it's working